community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/store/BatchingNeoStores.java

```diff
@@ -202,8 +201,7 @@ protected static Config getNeo4jConfig( Configuration config, Config dbConfig )
     {
         return new Config( stringMap( dbConfig.getParams(),
                 dense_node_threshold.name(), valueOf( config.denseNodeThreshold() ),
-                pagecache_memory.name(), valueOf( config.pageCacheMemory() ),
-                mapped_memory_page_size.name(), valueOf( config.pageSize() ) ),
+                pagecache_memory.name(), valueOf( config.pageCacheMemory() ) ),
```

Comment:
Just noticed the `config` object still has its `pageSize()` method. Can that be removed?